packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -169,14 +178,54 @@ class BuildModules {
     }, Promise.resolve([]));
   }
 
+  /**
+   * Find modules with locale directories to process
+   *
+   * @returns {SrcPkgDir[]} source package directories
+   */
+  async gatherModuleDirs() {
+    if (this._lookupModuleDirs) {
+      const promises = this._lookupModuleDirs.map(async lookupDir => {
+        const match = lookupDir.match(rePkgParts);
+        const pkgName = match.groups.name;
```

Comment:
Couldn't `match` end up being `null` if it doesn't conform to the pattern? Maybe a user-friendly error could be thrown in that case; I think it may just do a `TypeError` of some sort currently.

Reply:
Good catch - I updated the code and tests, include various name patterns addressing your [other comment](https://github.com/godaddy/gasket/pull/676#discussion_r1484864977).